src/main/webapp/enrichers/issueStateEnricher.js

```diff
@@ -0,0 +1,72 @@
+import { MessageEnricherBase } from 'symphony-integration-commons';
+import AssignUserService from '../services/AssignUserService';
+import CommentService from '../services/CommentService';
+
+const actions = require('../templates/actions.hbs');
+const errorDialog = require('../templates/errorDialog.hbs');
+
+const name = 'issueState-renderer';
+const messageEvents = ['com.symphony.integration.jira.event.v2.state'];
+
+export default class IssueStateEnricher extends MessageEnricherBase {
+  constructor() {
+    super(name, messageEvents);
+    this.implements = ['enrich', 'action', 'selected', 'changed'];
+
+    const assignUserService = new AssignUserService(name);
+    const commentService = new CommentService(name);
+
+    this.services = {
+      assignDialog: assignUserService,
+      assignIssue: assignUserService,
+      commentDialog: commentService,
+      commentIssue: commentService,
+      closeAssignDialog: assignUserService,
+      closeCommentDialog: commentService,
+    };
+  }
+
+  enrich(type, entity) {
+    const result = {
+      template: actions(),
+      data: {
```

Comment:
There are several object creations on the files. We could try to reduce this by creating a helper. [This link](https://stackoverflow.com/questions/19837916/creating-object-with-dynamic-keys) have a suggestion ;-)